ds/outputs/data/processed/cordis/sdg_01_project_count.csv

```diff
@@ -0,0 +1,8407 @@
+region_id,region_level,region_type,region_year_spec,sdg_01_project_count,year
```

Comment:
The header seems incorrect. It is:

```
region_id,region_level,region_type,region_year_spec,<id>,year
```

while it should be:

```
year,region_type,region_year_spec,region_id,region_level,<id>
```

Reply:
The headers match the schemas for these specific indicators. Do you mean that

```
year,region_type,region_year_spec,region_id,region_level,<id>
```

is the order that all indicators should follow (and that the schemas should also be adjusted accordingly)?

Reply:
Yes